api/converter_utils.go

```diff
@@ -401,6 +422,35 @@ func txnRowToTransaction(row idb.TxnRow) (generated.Transaction, error) {
 		localStateDelta = &d
 	}
 
+	var logs *[][]byte
+	if len(stxn.ApplyData.EvalDelta.Logs) > 0 {
+		l := make([][]byte, 0, len(stxn.ApplyData.EvalDelta.Logs))
+		for _, v := range stxn.ApplyData.EvalDelta.Logs {
+			l = append(l, []byte(v))
+		}
+		logs = &l
+	}
+
+	var inners *[]generated.Transaction
+	if len(stxn.ApplyData.EvalDelta.InnerTxns) > 0 {
+		itxns := make([]generated.Transaction, 0, len(stxn.ApplyData.EvalDelta.InnerTxns))
+		for _, t := range stxn.ApplyData.EvalDelta.InnerTxns {
+			extra2 := extra
+			// TODO: fix txn counter/number calculation and use ApplyData.ConfigAsset/ApplicationID
```

Comment:
the latest go-algorand has ApplyData.ConfigAsset/ApplicationID so newer transactions can use it for asset/app ids instead of calculating (both regular and inner)